docs/assets/snippet-styles.css

```diff
@@ -2,6 +2,20 @@
  * Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
  */
 
+:root {
+	/* This custom property is read by the JS and passed to editor configurations
+	as config.toolbar.viewportTopOffset. */
+	--ck-snippet-viewport-top-offset: 100
+}
+
+@media (max-width: 960px) {
+	:root {
+		/* The mobile layout of Umberto is different and the toolbar offset must be
+		smaller (https://github.com/ckeditor/ckeditor5/issues/1348). */
+		--ck-snippet-viewport-top-offset: 60;
```

Comment:
Value of this CSS Variable is wrong:

```
document.querySelector( '.header__main' ).offsetHeight
// 55
```

<img width="537" alt="screenshot 2018-11-20 at 12 07 53" src="https://user-images.githubusercontent.com/10219857/48769913-28a9f080-ecbd-11e8-9099-c3a0c2721df4.png">

Reply:
The value was correct for Umberto v0.11.x. I bumped Umberto to v0.12.x and changed the value.